docs/experimental-features/investigation-guide-actions.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[interactive-investigation-guides]]
+== Interactive investigation guides
+
+NOTE: This feature is available for {stack} versions 8.6.0 and newer.
+
+Detection rule investigation guides suggest steps for triaging, analyzing, and responding to potential security issues. For custom rules, you can create an interactive investigation guide that includes buttons for launching runtime queries in <<timelines-ui,Timeline>>, using alert data and hard-coded literal values. This allows you to launch detailed Timeline investigations directly from an alert using relevant data. 
+
+[role="screenshot"]
+image::images/ig-alert-flyout.png[Alert details flyout with interactive investigation guide,550]
+
+Each query button displays the number of documents found. Click the button to automatically create the query in Timeline based on the investigation guide's configuration.
```

Comment:
What are your thoughts on putting these two sentences in a separate section titled "Use investigation guides" or "Run investigation guides"? Currently, this information is bit hidden and assumes the user is reading the page from top to bottom. Moving it to a separate section could improve its visibility and searchability. It'd also cater to users who are just using the guides and not configuring them.

Reply:
I'm hesitant to break out a new section just for this info; maybe it'd be different if using the investigation guides was more complex than just clicking a button. @benironside suggested an [edit](https://github.com/elastic/security-docs/pull/2821#discussion_r1068250717) that might help make the distinction with configuration a little clearer.

Reply:
1+ to @benironside 's edit!